codebundles/twitter-query-tweets/sli.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Metadata          Author    Shea Stewart
+Documentation     Queries Twitter to count amount of tweets within a specified time range for a specific user handle.
+Force Tags        Twitter    Social  tweet
+Suite Setup       Suite Initialization
+Library           RW.Core
+Library           RW.SocialScrape.SocialScrape
+
+*** Keywords ***
+Suite Initialization
+    RW.Core.Import User Variable    HANDLE
+    ...    type=string
+    ...    description=The twitter handle to query.
+    ...    pattern=\w*
+    ...    example=gitbookstatus
+    RW.Core.Import User Variable    MAX_TWEETS
+    ...    type=int
+    ...    description=The number of the latest tweets to scrape.
+    ...    example=5
+    ...    default=5
+    RW.Core.Import User Variable    MAX_TWEET_AGE
+    ...    type=int
+    ...    description=The maximum age of the tweet in days.
+    ...    example=1
+    ...    default=1
+    RW.Core.Import User Variable    MIN_TWEET_AGE
+    ...    type=int
+    ...    description=The minimum age of the tweet in days.
+    ...    example=0
+    ...    default=0
+    Set Suite Variable    ${HANDLE}    ${HANDLE}
+    Set Suite Variable    ${MAX_TWEETS}    ${MAX_TWEETS}
+    Set Suite Variable    ${MAX_TWEET_AGE}    ${MAX_TWEET_AGE}
+    Set Suite Variable    ${MIN_TWEET_AGE}    ${MIN_TWEET_AGE}
+
+*** Tasks ***
+Query Twitter
+    ${rsp}=    RW.SocialScrape.SocialScrape.Twitter Scrape Handle    handle=${HANDLE}    maxTweets=${MAX_TWEETS}    max_tweet_age=${MAX_TWEET_AGE}    min_tweet_age=${MIN_TWEET_AGE}
+    ${metric} =    Get Length    ${rsp}
```

Comment:
`${metric} =` got an extra space hiding in there

Reply:
👍 eagle eye :) Thanks!